SocketRocket/SRWebSocket.m

```diff
@@ -373,6 +373,25 @@ - (void)_HTTPHeadersDidFinish;
 {
     NSInteger responseCode = CFHTTPMessageGetResponseStatusCode(_receivedHTTPHeaders);
     if (responseCode >= 400) {
+        
+        if (responseCode == 401)
```

Comment:
Funny enough, I was just thinking this last night. Made the change first thing this morning :)